portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,16 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomProject() {
```

Comment:
Add a JSDoc for this function, eg :
/**
  * Creates a new Circle from a diameter.
  * @param {number} d The desired diameter of the circle.
  * @return {Circle} The new Circle object.
  */

A more meaningful name such as showRandomProject instead of addRandomProject could be also considered.